modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -272,6 +273,45 @@ const makeRtcAdaptor = (tinymceEditor: Editor, rtcEditor: RtcRuntimeApi): RtcAda
   };
 };
 
+const makeNoopAdaptor = (): RtcAdaptor => ({
+  undoManager: {
+    beforeChange: () => {},
```

Comment:
`Fun.noop` instead here please and in all the places below (it'll minify better).

- [ ] Use `Fun.noop` instead of `() => {}`

Reply:
We should really setup a linter rule for this, so I'll add that to my tech debt list of things to look into.

Reply:
I totally forgot that these produce function and not the shorter lambdas but even with that we would save a few bytes. These will be fun to implement in reason since they would not type check if the arity is wrong so we would have to mix different noop<N> functions.